src/platform/forms-system/src/js/utilities/validations/index.js

```diff
@@ -136,3 +136,25 @@ export function isValidPartialMonthYearInPast(month, year) {
       momentDate.isSameOrBefore(moment().startOf('month')))
   );
 }
+
+/**
+ * Check validations for Custom pages
+ * @param {Function[]} validations - array of validation functions
+ * @param {*} data - field data passed to the validation function
+ * @param {*} fullData - full and appStateData passed to validation function
+ * @returns {String[]} - error messages
+ */
+export const checkValidations = (
+  validations = [],
+  data = {},
+  fullData = {},
+  index,
+) => {
+  const errors = { errorMessages: [] };
+  errors.addError = message => errors.errorMessages.push(message);
+  /* errors, fieldData, formData, schema, uiSchema, index, appStateData */
+  validations.map(validation =>
+    validation(errors, data, fullData, null, null, index, fullData),
```

Comment:
Can we rely on all validation functions to call `addError` on the errors array passed into it? If not, it might be worth calling that out somewhere. Just a thought

Reply:
Using `addError` is part of "normal" [form validation](https://depo-platform-documentation.scrollhelp.site/developer-docs/va-forms-library-getting-started-with-the-forms-li#VAFormsLibrary-GettingStartedwiththeFormsLibrary-Formvalidation)